components/landing/highlights.json

```diff
@@ -125,7 +125,7 @@
       "title": "Pervasive media censorship",
       "text": "Blocking of at least 121 news outlets",
       "report": "https://ooni.io/post/iran-internet-censorship/#news-websites",
-      "explore": "/search?input=bbc.co.uk&probe_cc=IR",
+      "explore": "/search?domain=bbc.co.uk&probe_cc=IR&test_name=web_connectivity",
```

Comment:
This query is missing date ranges. cc @agrabeli

Reply:
@sarathms why has "test_name=web_connectivity" been added here? Should I add it for all other queries as well?